python/large_test_files_script.py

```diff
@@ -40,6 +37,10 @@
 #Writes file to the output filepath and name in the custome file specifications yaml. 
 utils.write_file(path=output_filepath, ts_input=ts_data, lar_input=lar_data, name=output_filename)
 
+#If a row by row modification yaml file is present, the row by row modification function is applied. 
```

Comment:
Does this mean that if there is anything in the row-by-row yaml that any large files produced will have those characteristics?

Is it possible to make this more flexible so that a user can produce a stock large file or a large file with row-by-row without wiping the yaml?

Reply:
This part would only run if the user specified a yaml file in the configurations. Otherwise the row_by_row function would not be implemented.